lib/utils/createDomain.js

```diff
@@ -8,27 +8,27 @@ function createDomain(options, server) {
   // use location hostname and port by default in createSocketUrl
```

Comment:
`showStatus` now using its own logic, this is only used in client entries. I think this can be refactored in the future PR.

Reply:
I think logic for terminal output and for entries is different, it is similar in some places, but it is for different purpose